CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.14.6
+
+<!-- generated comparing v5.14.5..master -->
+
+_Aug 23, 2023_
+
+A big thanks to the 21 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Added the [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak
```

Comment:
@michaldudak It would be great to have a paragraph or two on this. For example, to share our plan when it comes to Popover vs. Popper vs. Popup. But also, when to use each, and lastly, why is there a new one.


I personally didn't capture the need for it. If I would try to guess it, it seems to be: since we are moving Popper to no longer use popper.js, we do no longer need to call this component Popper. We are renaming it to Popup to distinguish both versions between Material UI and Base UI, which keeping the API almost identical for easy migration.

I guess the alternative was to do a PopperV2, like GridV2.

Reply:
In https://github.com/mui/mui-x/issues/10006#issuecomment-1680523234 I felt that we could turn Popover to be a bit of a kitchen sink for creating popups, could support modal and non-modal modes, could be used by: select, autocomplete, menu, date pickers. And under the hood, would use Popup, Modal, and animations.

Reply:
I added an explanation on both the Popup and Popper docs pages, so hopefully, this won't be confusing. Here we can add:

```suggestion
- 🚀 Added the [Popup](https://mui.com/base-ui/react-popup/) component to Base UI (#37960) @michaldudak 
  It's intended to replace the Popper component, which uses the deprecated Popper JS library. The Popup is built on top of Floating UI and has a similar API to the Popper.
```